docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -75,10 +75,7 @@ This is particularly important when it comes to conformance and security.
 
 * Mandrel is recommended for building native executables that target Linux containerized environments.
 This means that Mandrel users are encouraged to use containers to build their native executables.
-If you are building native executables for macOS,
-you should consider using Oracle GraalVM instead,
-because Mandrel does not currently target this platform.
-Building native executables directly on bare metal Linux or Windows is possible,
+Building native executables directly on bare metal Linux, MacOS, or Windows is possible,
```

Comment:
That's fine, but we are still missing MacOS x86 binaries, so I suggest changing it as follows:

```suggestion
If you are building native executables for macOS on amd64/x86,
you should consider using Oracle GraalVM instead,
because Mandrel does not currently target this platform.
Building native executables directly on bare metal Linux, MacOS (on M processors), or Windows is possible,
```

or similar

Reply:
Thanks! I wasn't sure of exactly what worked, but I totally forgot about intel Mac.